### DISCUSSION TITLE
Visual mode doesnt work

### DISCUSSION THREAD

**Discussion by teckter312** [seerge/g-helper#3314] [Q&A]
After the v0.197 update the visual mode doesnt work anymore , it doesnt do anything after i change the mode. Anyone have the same problem ?

**Comment by seerge** [comment]
@teckter312 helo,

Visual Modes are handled by AsusSplendid utility (that AC or G-Helper call). And there is nothing changed in this regard in 0.197 (just one more new `Disabled` mode is added).

Does older version makes any difference ? Please upload log from `%appdata%\GHelper`.

**Reply by teckter312** [reply]
this one is from the new version 
[log.txt](https://github.com/user-attachments/files/17534974/log.txt)

**Reply by seerge** [reply]
@teckter312 try to manually open `C:\ProgramData\ASUS\GameVisual` and delete all files there. Then restart G-Helper and re-download color profiles that it will offer

**Reply by teckter312** [reply]
i try other version to it didnt work eather and then i try to disable my nvidia gpu and then the visual mode work, maybe something with the new nvidia driver and not ghelper problem.

**Reply by seerge** [reply]
@teckter312 it could be that Nvidia is blocking something that AsusSplendid is trying to do. 
Did you try to delete and re-download color profiles as I proposed ?

**Reply by teckter312** [reply]
yes i did

**Reply by seerge** [reply]
You can also try to open `C:\ProgramData\ASUS\ASUS System Control Interface\AsusOptimization\Splendid` and delete (or just move somewhere) a sys file in that folder with your username. It's a settings of AsusSplendid.

They will be re-created next time you try to set Visual Mode

**Reply by teckter312** [reply]
it still doesnt work either.

**Reply by seerge** [reply]
@teckter312 ok, may be try to reinstall latest drivers from Nvidia website (and tick Factory Defaults) during reinstall ? (but don't forget to enable GPU before that of course)

**Reply by teckter312** [reply]
the first time i forgot to switch to the nvidia gpu after the reinstall so it work, after i switch to the gpu it doesnt work anymore.

**Reply by seerge** [reply]
Do you use latest Asus System Control Interface driver (AsusSplendid is being updated along it). You can check for it under `Updates` section in G-Helper.

**Reply by teckter312** [reply]
yeah is on the latest version 
it work both on eco and ultimate mode but not when i use strandard mode and switch to the dgpu using nvidia advanced optimus.

**Reply by seerge** [reply]
@teckter312 then it's probably some hiccup in AsusSplendid itself that can't handle Advanced Optimus. As Optimus creates a second (virtual) display in windows, that's probably confusing Asus utility.
